scripts/hudson/narayana.sh

```diff
@@ -367,7 +338,7 @@ function lra_tests {
 
 
 function enable_as_trace {
-    CONF=${1:-"${JBOSS_HOME}/standalone/configuration/standalone-xts.xml"}
+    CONF=${1:-"${JBOSS_HOME}/standalone/configuration/standalone.xml"}
```

Comment:
yeah, I think this should be a separate commit as Tom said above.

Reply:
Thanks, I will create a separate commit for that.